extensions/markdown-language-features/src/markdownEngine.ts

```diff
@@ -209,6 +209,15 @@ export class MarkdownEngine {
 	}
 
 	private addImageStabilizer(md: any): void {
+		md.core.ruler.push(
```

Comment:
Instead of adding a new rule, can we update the other rules to check that `env` is defined before trying to access properties on it?

Reply:
Thanks for the recommendation
My most recent change should technically do it.